sdk/tests/conformance/textures/misc/exif-orientation.html

```diff
@@ -93,8 +78,14 @@
     let blob = await response.blob();
     output("----------------------------------------------------------------");
     output("Testing " + filename + " via ImageBitmap from Blob");
-    await testImageBitmapFromBlobWithFlipY(blob, true);
-    await testImageBitmapFromBlobWithFlipY(blob, false);
+    await testImageBitmapWithFlipY(blob, true);
```

Comment:
in these you would do
await testImageBitmapWithFlipY(blob, undefined);
await testImageBitmapWithFlipY(blob, "flipY");
await testImageBitmapWithFlipY(blob, "none");

Reply:
Done.